src/styles/_mixins.scss

```diff
@@ -0,0 +1,94 @@
+// This is where you put all your functions that create SCSS for you
```

Comment:
what's the pattern for using `_` as a prefix for these filenames? does it signify something?

Reply:
It just means that it's a fragment, so that when it gets compiled, it doesn't get compiled into an actual css file